aiohttp/web_urldispatcher.py

```diff
@@ -1121,3 +1115,18 @@ def add_routes(self,
         for route_def in routes:
             registered_routes.extend(route_def.register(self))
         return registered_routes
+
+
+def _quote_path(value: str) -> str:
+    return URL.build(path=value).raw_path
```

Comment:
It might be cleaner to explicitly pass `encoded=False` here, to make clear what we're trying to do, and contrast it with `_unquote_path` below.